module/Photo/src/Photo/Controller/PhotoController.php

```diff
@@ -14,8 +14,35 @@ public function indexAction()
         $albums = $album_service->getAlbums();
         $photo_service = $this->getPhotoService();
 
+        //we'll fix this ugly thing later vv
+        $config = $this->getPhotoService()->getConfig();
+        $basedir = str_replace("public", "", $config['upload_dir']);
+        $albums = $photo_service->populateCoverPhotos($albums);
```

Comment:
De populateCoverPhotos functie is niet echt de ideale oplossing hier. Als er geen cover foto ingesteld is voor een album, dan is deze null. In dit geval moet er een random foto uit het album als cover foto genomen worden. Momenteel wordt dit rerealiseerd door populateCoverPhotos. Suggesties voor een betere oplossing zijn welkom.